etc/afpd/dircache.c

```diff
@@ -241,6 +247,8 @@ static int hash_comp_didname(const void *k1, const void *k2)
 
 static q_t *index_queue;    /* the index itself */
 static unsigned long queue_count;
+static unsigned long queue_count_max =
+    0;  /* Peak entries reached this session */
```

Comment:
nitpick: rather than letting the code comment break the line after the equal sign, please put the comment above the line, or remove it altogether.

FWIW, inserting this kind of line break when a trailing code comment exceeds the line length limit is a bad habit of astyle, and I filed a feature request with the project but alas no reaction yet

https://gitlab.com/saalen/astyle/-/issues/93